src/components/common/Actions.vue

```diff
@@ -147,10 +146,9 @@ export default class Actions extends Mixins(DateMixin, FilingTemplateMixin, Name
     this.setIsFilingPaying(true)
 
     // If this is a named company IA, validate NR before filing submission. This method is different
-    // from the processNameRequest method in App.vue. This method shows a generic message if
-    // the Name Request is not valid and clicking ok in the pop up redirects to the Manage Businesses
-    // dashboard.
-    if (this.isNamedBusiness && this.hasNewNr) {
```

Comment:
"is named business" is nebulous at the moment -- we can't tell for sure in an IA.

Instead, checked if there is a name request number.

This will be sorted out further in [# 13022](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/13022).